Makefile

```diff
@@ -60,22 +60,42 @@ gendocs:
 
 # if want to add all the test add the tag --tags never at the end
 #	ansible-test integration $(TEST_ARGS) --tags never
-integration-test: create-integration-config
+integration-test: create-integration-config create-e2e-firewall
+	@echo "Running Integration Test(s)..."
 	ansible-test integration $(TEST_ARGS)
 
-test: integration-test
+create-e2e-firewall:
+	@echo "Running create e2e firewall playbook..."
+	@if ansible-playbook scripts/create_e2e_cloud_firewall.yaml > /dev/null; then \
+		echo "Successfully created e2e firewall"; \
+	else \
+		echo "Failed to create e2e firewall"; \
+	fi
+
+delete-e2e-firewall:
+	@echo "Running delete e2e firewall playbook..."
+	@if ansible-playbook scripts/delete_e2e_cloud_firewall.yaml > /dev/null; then \
+		echo "Successfully deleted e2e firewall"; \
+	else \
+		echo "Failed to delete e2e firewall"; \
+    fi
+
+test: integration-test delete-e2e-firewall
 
 testall: create-integration-config
 	./scripts/test_all.sh
```

Comment:
I think we use `make testall` to run the tests in CI. Should we also make it create firewall?